src/read/macho/dyld_cache.rs

```diff
@@ -96,11 +593,11 @@ where
             };
             let uuids = v1.iter().map(|e| &e.uuid).chain(v2.iter().map(|e| &e.uuid));
             for (&data, uuid) in subcache_data.iter().zip(uuids) {
-                let sc_header = macho::DyldCacheHeader::<E>::parse(data)?;
-                if &sc_header.uuid != uuid {
+                let header = macho::DyldCacheHeader::<E>::parse(data)?;
+                if &header.uuid != uuid {
```

Comment:
Why change these?

Reply:
It just seems neater when the variable is only used once. Can revert if you'd like?